app/views/posts/edit.html.slim

```diff
@@ -0,0 +1,8 @@
+= form_with model:@post do |f|
```

Comment:
Šeit tev atnaca laiks izmantot tadu lietu kā partials.
https://guides.rubyonrails.org/layouts_and_rendering.html#using-partials

Tev vajadzetu izveidot jaunu partial posts/_form.html.slim
Lai nedublēt kodu. Un tad renderet to iekš edit un new.

Un taja jau norādit ar condition, kam vajadzetu rendereties un kam nav, pie kada aktiona.
To tu vari izdarit ar controller_action vai action_name, tā no galvas neaceros:
```ruby
- if action_name == 'edit'
   = link_to "Delete" , post_path(@post.id), data: { confirm: "Are you sure?" }, method: :delete, class: "btn btn-danger"
```

Reply:
Vel lūdzu atsrpi model: @post